plugin/mutate/source/dextool/plugin/mutate/backend/analyze/package.d

```diff
@@ -964,9 +964,18 @@ struct Analyze {
     }
 
     private {
-        static immutable rawReNomut = `^((//)|(/\*))\s*NOMUT\s*(\((?P<tag>.*)\))?\s*((?P<comment>.*)\*/|(?P<comment>.*))?`;
+        static immutable rawReNomut = `^((//)|(/\*))\s*NOMUT\s*(\((?P<tag>.*)\))?\s*((?P<comment>.*)\*/|(?P<comment>.*))?`; 
```

Comment:
I think you can get away with one regex by modifying `rawReNomut`.
something like this:

```d
rawReNomut = `^((//)|(/\*))\s*NOMUT(?P<type>\w*)\s*(\((?P<tag>.*)\))?\s*((?P<comment>.*)\*/|(?P<comment>.*))?`;
```

then you can check if type is BEGIN, END, NEXT or empty